client/js/render.js

```diff
@@ -1,12 +1,15 @@
 const hyperHTML = require('hyperhtml');
 
+const asyncRenderView = (viewName, wire, model) =>
+  import(`../../shared/view/${viewName}.js`).then(renderFn => renderFn(wire, model));
```

Comment:
So this is a clever feature of webpack called ContextModules.

We basically can treat a "context" (aka directory), as a module. What this allows is for webpack to resolve every module that matches that pattern in that partial path, and create separate async bundles out of each of those modules. This supports "pseudo" dynamic (via static analysis) loading.